test/utils/naming.test.js

```diff
@@ -47,4 +49,15 @@ describe('Given naming file', () => {
       expect(naming.addSuffix(label, environment)).toEqual('Label - testProject-testWorkspace')
     })
   })
+  describe('When getRegistrationName is called', () => {
```

Comment:
I would try using `test.each`, first you can setup the cases by iterating over the collections and joining them into a single collection. Then you can using the parameters to construct a dynamic test name, which is easier to debug if something fails.
```suggestion
  describe('When getRegistrationName is called', () => {
    const cases = Object.keys(events).flatMap(entityName => 
      providers.map(({key: providerKey}) => ({
          entityName,
          providerKey
      }))
    );

    test.each(cases)("naming.getRegistrationName($providerKey, $entityName) returns strings with less than 25 chars", ({entityName, providerKey}) => {
      const result = naming.getRegistrationName(providerKey, entityName)
      expect(result.length).toBeLessThanOrEqual(25)
    })
  })
```

Reply:
Done in https://github.com/adobe/commerce-integration-starter-kit/pull/15/commits/804bc15c72d8738fe349aeeee0db5e9b568bcf86